milli/src/search/mod.rs

```diff
@@ -446,6 +451,27 @@ mod test {
     use super::*;
     use crate::index::tests::TempIndex;
 
+    #[test]
+    fn test_kanji_language_detection() {
```

Comment:
```suggestion
    #[cfg(feature = "default")]
    #[test]
    fn test_kanji_language_detection() {
```

Not sure, but shouldn't we add this feature flag?